Sources/SpeziContact/Models/Contact.swift

```diff
@@ -11,6 +11,63 @@ import SwiftUI
 
 
 /// Encodes the contact information.
+///
+/// ### Usage
+///
+/// The following example demonstrates the usage of the ``Contact`` in a ``ContactsList``.
+/// ```swift
+/// struct Contacts: View {
+///     let contacts = [
+///         Contact(
+///             name: PersonNameComponents(
+///                 givenName: "First",
+///                 familyName: "Last"
+///             ),
+///             image: Image(systemName: "figure.wave.circle"),
+///             title: "Title",
+///             description: "Description",
+///             organization: "Organization",
+///             address: {
+///                 let address = CNMutablePostalAddress()
+///                 address.country = "USA"
+///                 address.state = "CA"
+///                 address.postalCode = "94305"
+///                 address.city = "City"
+///                 address.street = "123 ABC St"
+///                 return address
+///             }(),
+///             contactOptions: [
+///                 .call("+1 (123) 456-7890"),
+///                 .text("+1 (123) 456-7890"),
+///                 .email(addresses: ["email@address.com"]),
+///             ]
+///         )
+///     ]
+/// }
+/// ```
+///
+/// Here is an additional example of a view  (`ContactRow`), that takes a single ``Contact`` and displays their
```

Comment:
Please combine the two code examples (so the one example in this line and the one above)